### ISSUE TITLE
Add GF check: Do Stylistic Sets have names?

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#3695]
## Observed behaviour

The [GlyphsApp Tutorial for Stylistic Sets Naming](https://glyphsapp.com/learn/stylistic-sets#g-__names-for-stylistic-sets) explains how to set labels for OpenType Feature Stylistic Sets. 

## Expected behaviour

Each stylistic set should be named, in every GF font; therefore we should have a FB GF profile check if these names exist.

**Comment by felipesanches** [comment]
This is definitely not the same issue, but perhaps this would lead to similar considerations as the things discussed at #3155

**Comment by vv-monsalve** [comment]
There is already a check for this and a discussion around it #3155 

It was downgraded to a "Warn" after a long discussion there.

**Comment by davelab6** [comment]
Oh right, yes. I think it should be MUST for GF profile, and I filed this issue forgetting we already made a check for this. 

The discussion proposed to make it MUST in the GF profile after better docs were made to explain how to make the check pass. I'd like to make this a Q2 goal as I think its essential for GF to "go big" with OpenType Features that all GF fonts have labels for all OpenType features in the TTFs, so that the fonts.google.com catalog UI can pull them through to users, etc.
